torch/_higher_order_ops/wrap.py

```diff
@@ -64,17 +69,11 @@ class TagActivationCheckpoint(HigherOrderOperator):
     This operator is supposed to be used only with torch.compile stack. This
     accepts a Fx graph module which needs to be checkpointed. This operator adds
     "recomputable" tag to the nodes of the Fx graph that should be recomputed.
-
-    The goal is to avoid both Dynamo and AOT Autograd to trace through saved
-    tensor hooks, and rather rely on the partitioners to actually duplicate the
-    nodes. This sits well in the torch.compile stack, because by the time graph
-    reaches partitioner, inductor has already run its functionalization of rng
-    ops. Therefore, the duplication of nodes, by design, respects the rng states
-    in the forward and recomputed forward in backward.
```

Comment:
I think this is still partially true.

Dynamo still does not trace through saved tensor hooks.

AOT Autograd does trace through saved tensor hooks but has special logic with TorchDispatchMode to override the usual saved_tenso_hooks fn to tag the nodes.

The comment about RNG is still valid.